libs/block-editor/src/lib/extensions/blocks/image-block/image-block.extention.ts

```diff
@@ -15,16 +30,36 @@ export const ImageBlock = (injector: Injector): Node => {
             return {
                 data: {
                     default: null,
-                    parseHTML: (element) => {
-                        data: element.getAttribute('data');
-                    },
+                    parseHTML: (element) => element.getAttribute('data'),
                     renderHTML: (attributes) => {
                         return { data: attributes.data };
```

Comment:
We have this issues https://github.com/dotCMS/core/issues/21509 for the contentlet block. Can you please check if this is not happening in the image block?